xblockmufi/public/edit.js

```diff
@@ -32,37 +41,39 @@ function XblockMufiEdit(runtime, element) {
             },
             error: function buttonSaveOnError() {
                 runtime.notify('error', {});
-            }
+            },
         });
+        /* eslint-enable camelcase */
         return false;
     });
 
     // TEXT EDITOR JS
-    $('li.specialButton').click(function(e) {
+    /* eslint-disable no-invalid-this*/
+    $('li.specialButton').click(function (e) {
         e.preventDefault();
         $(this).toggleClass('active');
         $('.sr .toggle-open', this).toggle();
         $('.specialCharTable', $element).toggle();
     });
 
-    $('.txtEditor li.styleButton').mousedown(function(e){
+    $('.txtEditor li.styleButton').click(function (e) {
```

Comment:
I think I had made this `mousedown` so that the text area would remain in focus?

Reply:
FWIW, I think the linter did this automatically;
I don't think I did it manually.

Can you see if the UI/UX is degraded?

Reply:
Yeah, I just tried making the change in my workbench, that's exactly why I did that.

With `mousedown` it keeps the cursor exactly where it was.

I think we should change it both in this file and `view.js`

Reply:
mousedown here too, this is used by instructors to layout the transcription for comparison with the learners.

Reply:
Done.